src/v2/service.rs

```diff
@@ -305,7 +305,9 @@ impl Service {
         let mut new_env = BTreeMap::new();
         for rel_path in &self.env_files {
             let env_file = EnvFile::load(&base.join(&rel_path.value()?))?;
-            new_env.append(&mut env_file.to_environment()?);
+            new_env.extend(env_file.to_environment()?
+                           .into_iter()
+                           .map(|(k, v)| (k, Some(v))));
```

Comment:
If environments now have type `BTreeMap<String, Option<RawOr<String>>>`, then we should consider updating `to_environment` to also return that type, instead of fixing it here.

Reply:
Turns out env_files also accept valueless keys, so we should just handle them there and produce the same `BTreeMap` type.